extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -8,7 +8,6 @@ class ArticleAsJson extends WikiaService {
 	];
 
 	const CACHE_VERSION = '0.0.3';
-	const CACHE_VERSION_FOR_SEO_FRIENDLY_IMAGES = 1; // for $wgEnableSeoFriendlyImagesForMobile
```

Comment:
I don't think so. The code that I removed wasn't used for few weeks now. Do you think it's possible that something is still cached since then?